pkg/api/v1/auxpb/aux_service.pb.go

```diff
@@ -1,7 +1,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
 // 	protoc-gen-go v1.22.0
-// 	protoc        v3.11.4
```

Comment:
This should be fine, and upgrading to `3.11.2` in alpine would require using `edge`.